<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>editPrefSkel</class>
 <widget class="QWidget" name="editPrefSkel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>668</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Preferences</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTreeView" name="prefTreeView"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
